tests/ConstructAndDestroyComponentTests.js

```diff
@@ -10,6 +10,10 @@ https://raw.githubusercontent.com/fluid-project/infusion-nexus/master/LICENSE.tx
 
 /* eslint-env node */
 
+// TODO: test that PUT responses *do not* have JSON headers
+//       in fact, anything with a non-extant response body should not have a content type
+// Current issue is that I need a workable way to specify that particular values are undefined
```

Comment:
Looks like you have done this? This bit of comment can be removed, is the PUT work still outstanding?

Reply:
Good spot. I did indeed resolve this.